extensions/sonarqube/vss-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "sonarqube",
   "name": "SonarQube",
-  "version": "5.3.0",
```

Comment:
Shouldn't this be a bugfix release?

Reply:
I think the bugfix release will be the one once we really fix the scanner, not?

Reply:
BTW If we wait like an hour or to, the scanner bugfix release might be done and we could embed that version of the scanner right away?

Reply:
Yes you're right, i'm fixing it.

Reply:
@csaba-sagi-sonarsource I would rather mitigate the issue ASAP and we can do the bugfix release of the scanner also ASAP but this is affecting users as we speak so 1h matters

Reply:
We just have to think of the cache that Azure Pipelines apply on the extensions : if there are too many release in a short period of time, it takes time before it's actually rolled out everywhere.

Reply:
And many users cannot downgrade so are blocked

Reply:
There is no hurry for the AZDO 2nd release, and it should only be done in a morning (we need a postportem for this).

I meant that we need to do the bugfix release for the S4NET , for the people manually using the S4NET in their pipelines.